### PR TITLE
#243 - remove references to H2 as it is no longer supported.  Up'ed J…

### PR DESCRIPTION
**Pull request by rmmayo**
…DK minimum version to 17

### CODE REVIEW

File: docs/dashboard/install-guide/quickStart.md [hunk 1]
```diff
@@ -46,13 +63,20 @@ curl -s https://api.github.com/repos/NationalSecurityAgency/skills-service/relea
...
+-Dspring.datasource.password=skillsPassword \
+-jar ~/Downloads/skills-service-X.X.X.jar
 ```
 
 <import-content path="/dashboard/install-guide/common/service-install-output-and-backend.html"/>
 
 ## 3. Integrate Client Libraries
 
+::: tip Please Note
+Please note that the integrating the SkillTree Client Libraries is **optional** and is only applicable when integrating gamification training directly into an existing or new web application.
```
Comment (dwalizer) [inline docs/dashboard/install-guide/quickStart.md:77]: Should probably be `Please note that integrating`

File: docs/contribution/devEnv.md [hunk 2]
```diff
@@ -236,7 +240,7 @@ SkillTree's overall testing strategy is to implement black-box integration tests
 
 We like tests (especially integration tests), so please make sure you thoroughly test your code.  
 Prior to making a Pull Request make sure that ALL tests pass:
-1. Run services tests against PostgreSQL in addition to the default in-memory H2 database
+1. Run all services tests
```
Comment (dwalizer) [inline docs/contribution/devEnv.md:243]: Might want to change `services` to `service`

Comment (rmmayo) [reply docs/contribution/devEnv.md]: :+1:

File: docs/dashboard/install-guide/quickStart.md (hunk 1 above)
Comment (rmmayo) [reply docs/dashboard/install-guide/quickStart.md]: :+1:

**Review by sudo-may — APPROVED** [review]

**Pull request merged**